internal/server/v2/event/golden/event_collection_date/EarthquakeEvent_CA.json

```diff
@@ -0,0 +1,1197 @@
+{
```

Comment:
Can this just return a list?

Reply:
Similar to "eventCollection", the motivation to have a wrapper like this that we can have a "bulk" method in the future.

Currently in V1, EventCollectionDate is not a "bulk" method, and we were planning to add a "bulk" method, that's why we have this wrapper, so that in the future, "eventCollectionDate" can be a repeated field.